src/main/java/dev/aziz/movies/controllers/MovieController.java

```diff
@@ -37,8 +37,8 @@ public ResponseEntity<MovieDto> getMovie(@PathVariable Long id) {
     }
 
     @PostMapping("/movies")
-    public ResponseEntity<MovieDto> createMovie(@RequestBody @Valid MovieDto movieDto) {
-        return ResponseEntity.created(URI.create("/movies/" + movieDto.getId())).body(movieService.createMovie(movieDto));
+    public ResponseEntity<MovieDto> createMovie(@RequestBody @Valid Movie movie) {
```

Comment:
Don't use the entities in the controllers, use the DTOs and use a mapper in the Service to transform an entity into a DTO.